### PR TITLE
1584 redirect placeholder page

### LINKED ISSUE #1584: Create placeholder page for https://reviewer.elifesciences.org/redirect
## Problem / Motivation
- WHO wants this? Authors and product
- WHEN and WHERE do they want it? This page will form part of the journey for users who attempt to begin a new submission in eJP and need to be redirected after we are supporting 100% of new submissions through xPub.
- WHAT do they want? Information explaining that they are being redirected and why
- WHY do they want this? We were unable to implement the redirection on submit.elifesciences.org as originally planned, so there may still be some users attempting to begin new initial submissions in eJP. This interstitial will provide enough context to support this less than desirable user journey. 

## Proposed solution
The placeholder page should be created initially, and then content added

## Clarification needed and assumptions
eJP will point their redirection to this page in production

---

## Tasks
- [ ] A description of a task
- [ ] A description

## Technical notes
- We could use this technology
- More notes or suggestions

### LINKED ISSUE #1613: Remove header on https://reviewer.elifesciences.org/redirect
We currently have a generic layout which get's applied to all of our app pages. This should not be used for the LandingRedirect page.

https://github.com/elifesciences/elife-xpub/blob/develop/app/routes.js#L20

### PR DESCRIPTION
**Pull request by hem-brahmbhatt**
#### Background

- Creates a placeholder page for the `/redirect` route. Page is ready for content to be added.
- Added a browser test
- Deleted unnecessary config mocking

#### Any relevant tickets

Closes #1584 
Closes #1613 

#### How has this been tested?
**Reviewers** ensure that these test requirements are also reviewed.

Please indicate the need for this change to be tested. If not please justify, if so then at what levels. 

- [ ] Unit / Integration tests
- [ ] Browser tests
- [ ] End2end tests

### CODE REVIEW

**Comment by diversemix** [comment]
@damnhipster ...
```
[2019-02-27T16:20:16.605Z] /home/xpub/app/components/pages/LandingRedirect/LandingRedirect.js

[2019-02-27T16:20:16.605Z]    4:1   error  Component should be written as a pure function  react/prefer-stateless-function

[2019-02-27T16:20:16.605Z]    5:3   error  Useless constructor                             no-useless-constructor

[2019-02-27T16:20:16.605Z]   11:13  error  HTML entities must be escaped                   react/no-unescaped-entities

```

**Review by diversemix — APPROVED** [review]

**Comment by diversemix** [comment]
@damnhipster ... hope you don't mind I just disabled the linting rules in this file as this is a work in progress

**Review by jyenes — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -59,7 +59,7 @@
     ]
   },
   "jest": {
-    "collectCoverage": true,
+    "collectCoverage": false,
```
Comment (Cooryd) [inline package.json]: Was this pushed accidentally?

Comment (hem-brahmbhatt) [reply package.json]: 🤦‍♂️

**Pull request merged**